### PR TITLE
introduce project and explain notification_slack

### PR DESCRIPTION
**Pull request by jrit**
@dmaloneycalu readme updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,6 +282,12 @@ This list aims to be a reference and may not cover every detail of our implement
 
   Because this task is at the end of a `deploy`, it can fail without preventing deployment. This will show as a failure in the `terminal` and CI services but the files have been uploaded to AWS. **IMPORTANT**: The notification task will fail when AWS SES is in sandbox mode and any of the recipient emails are not verified.
 
+
+- #### notification_slack
+  A task to send an message to a slack channel after a deployment completes.
```
Comment (dmaloneycalu) [inline README.md]: "send a message"

**Comment by dmaloneycalu** [comment]
One typo, and bump the package patch since we've already released v6.1

**Comment by jrit** [comment]
You think we need to do a release for this update? I was just going to wait until the next code change.

**Comment by dmaloneycalu** [comment]
Not really I suppose.

**Pull request merged**